.github/ISSUE_TEMPLATE/06_localization_data_issue.yml

```diff
@@ -0,0 +1,104 @@
+        ## Some recommended best practices.
+        Due to how localization data is handled, there are some best practices you should keep in mind while designing applications and writing tests;
+        * For any server application (and extending to most persistent backend local services), you _SHOULD_ run the process with an invariant-like culture set (eg, `LC_ALL=C.UTF-8` on Linux), and in the UTC timezone.  For resiliency, such applications _SHOULD_ still explicitly provide the [`InvariantCulture`](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-8.0) when internally formatting data, and _SHOULD_ also still use UTC-sourced data (such as [`TimeProvider.GetUtcNow()`](https://learn.microsoft.com/en-us/dotnet/api/system.timeprovider.getutcnow?view=net-8.0#system-timeprovider-getutcnow)).
+        * For any local user-facing (most often GUI-based) application, you **MUST** respect and use the culture of the user.
+        * When practicable, you _SHOULD_ consider using and enabling format overrides.  On Windows this can be achieved just by calling the proper [`CultureInfo()` constructor](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.-ctor?view=net-8.0#system-globalization-cultureinfo-ctor(system-string-system-boolean)).  Other operating systems or web applications may require custom code.
+        * For applications that are distributed, allow zero-downtime updates (eg, Azure Function Applications), or similar you **MUST PLAN** for situations where there may be multiple instances of the application with different format or other culture data (including timezone rules).
+        * For any element being used for interchange (eg, a field in a JSON document in an API call) or storage, you **MUST** use the [`InvariantCulture`](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-8.0); additionally, in many cases you _SHOULD_ either use a roundtrip format (such as the [`"O"`, `"o"` date-time format specifier](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier)), or explicitly specify a format string.  Although the format strings for the invariant culture are stable and unlikely to change, specifying an explicit format string helps to clarify the intent of your code.
+          * For date/time elements, consider the advice and observations in Jon Skeet's post [_Storing UTC is not a silver bullet_](https://codeblog.jonskeet.uk/2019/03/27/storing-utc-is-not-a-silver-bullet/).
```

Comment:
> or explicitly specify a format string

I think this can still cause a problem if using formats produce culture specific terms (e.g. month names) which will break parsing if other culture is used.